collect_app/src/main/java/org/odk/collect/android/application/FeatureFlags.java

```diff
@@ -0,0 +1,10 @@
+package org.odk.collect.android.application;
+
+public class FeatureFlags {
+
+    public static final boolean STATIC_ADD_REPEAT_LANGUAGE = true;
```

Comment:
This is just using a very simple static feature flag. If we were looking at having tests for both behaviours I'd generally look at having a `FeatureFlag` object that could be passed to objects that need it.